vector/src/main/java/im/vector/adapters/VectorMessagesAdapter.java

```diff
@@ -215,26 +214,34 @@ public VectorMessagesAdapter(MXSession session, Context context, MXMediasCache m
                 R.layout.adapter_item_vector_message_room_member,
                 R.layout.adapter_item_vector_message_text_emote_notice,
                 R.layout.adapter_item_vector_message_file,
-                R.layout.adapter_item_vector_message_image_video,
                 R.layout.adapter_item_vector_message_merge,
+                R.layout.adapter_item_vector_message_image_video,
```

Comment:
What did you have move `adapter_item_vector_message_image_video`?
It now appears twice.